vignettes/utf8.Rmd

```diff
@@ -475,3 +475,11 @@ the [readtext][readtext] package.
 [utf8]: https://en.wikipedia.org/wiki/UTF-8
 
 [windows1252]: https://en.wikipedia.org/wiki/Windows-1252
+
+Session information
+-------
+
+```{r}
+sessioninfo::session_info(include_base = TRUE)
```

Comment:
Happy to also convert this to use `utils:: sessionInfo()` if you'd rather avoid a new soft dependency.